x-pack/plugins/endpoint/public/applications/endpoint/store/test_utils.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Dispatch } from 'redux';
+import { AppAction, GlobalState, MiddlewareFactory } from '../types';
+
+/**
+ * Utilities for testing Redux middleware
+ */
+export interface MiddlewareActionSpyHelper<S = GlobalState> {
```

Comment:
Moved this test utility under `/store` since its not specific to Host list and can be used by any of the test cases. Currently used in the `mocks/app_context_render`